styleguide/source/_patterns/03-organisms/ribbon.json

```diff
@@ -45,13 +45,13 @@
       "triggerText": "",
       "unorderedList": [
         {
-          "text": "<a href=\"#\">Become a Member</a>"
+          "text": "<a class=\"ama__link--white\" href=\"#\">Become a Member</a>"
         },
         {
-          "text": "<a href=\"#\">Sign In</a>"
+          "text": "<a class=\"ama__link--white\" href=\"#\">Sign In</a>"
         },
         {
-          "text": "<a href=\"#\">Create Account</a>"
+          "text": "<a class=\"ama__link--white\" href=\"#\">Create Account</a>"
```

Comment:
@designerneil instead of adding the class to every items in the json can we `@extend` it in the `.ama__ribbon__dropdown__nav__menu a` which already exists in `_ribbon-dropdown.scss`?

This definitely solves the problem, I am concerned that it will become an issue again in Drupal of making sure the data is in the right place. Ideally, this should be consistent throughout the code.